### PR TITLE
Created basic ui accordingly

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
- Created a app.rb file that serves as console app entry-point. It have methods that do the following:
List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.
In main.rb defined the entry point, this is a method called main that is invoked at the end of the file. This method does the following:
Present the user with a list of options to perform.
Lets users choose an option.
If needed, ask for parameters for the option.
Have a way to quit the app.
Thanks

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  end
+
+  def list_books
+    if books.empty?
+      puts 'No book created'
+      return
+    end
+    puts "Amount of books #{books.length}"
+    books.each_with_index do |b, index|
+      puts b[id]
```
Comment (Whoistolu) [inline app.rb]: - [ ] [REQUIRED] After creating a book, I am unable to list the books created. I get the error bellow:
Screenshot 2023-04-14 at 21 58 11

### My Observation
The error message you're seeing suggests that the list_books method is looking for a variable called `id` that hasn't been defined or initialized.

Let's define `id` to get this code to work 💪🏽

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @sajibarafatsiddiqui 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply app.rb]: Thanks. fixed that issue.

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @sajibarafatsiddiqui, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**